src/main/java/com/google/sps/servlets/CreateCalendarServlet.java

```diff
@@ -86,13 +86,8 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
 
     String currentUserEmail = UserServiceFactory.getUserService().getCurrentUser().getEmail();
     BatchRequest batch = calendarService.batch();
-    
-    AclRule rule = new AclRule();
-    Scope scope = new Scope();
-    scope.setType("default").setValue("");
-    rule.setScope(scope).setRole("reader");
 
-    Insert insertRequest = calendarService.acl().insert(createdCalendar.getId(), rule);
+    Insert insertRequest = Utils.createUserAclRequest(createdCalendar.getId(), "", "default", "reader");
```

Comment:
What does this line do? Does it make the calendar public? In that case, add a comment before the line to describe that since its not really clear from the code.

Reply:
resolved